scripts/Inelastic/vesuvio/commands.py

```diff
@@ -41,17 +42,31 @@ def fit_tof(runs, flags, iterations=1, convergence_threshold=None):
     # Load
     spectra = flags['spectra']
     fit_mode = flags['fit_mode']
-    sample_data = load_and_crop_data(runs, spectra, flags['ip_file'],
-                                     flags['diff_mode'], fit_mode,
-                                     flags.get('bin_parameters', None))
+    flags['runs'] = str(runs)
```

Comment:
Did you mean to wrap this in a `str` here? If `runs` is a workspace this will not be very useful.

Reply:
@martyngigg flags['runs'] should be a str - and the runs parameter here shouldn't be. This is because flags['runs'] is later used in the creation of the names of the output. If this is a workspace it should use the workspace name, else it should use the given runs input.

Reply:
:+1: